pandas/tests/io/excel/test_writers.py

```diff
@@ -751,7 +751,8 @@ def test_to_excel_timedelta(self, path):
         tm.assert_frame_equal(expected, recons)
 
     def test_to_excel_periodindex(self, tsframe, path):
-        xp = tsframe.resample("ME", kind="period").mean()
+        # xp has a PeriodIndex
+        xp = tsframe.resample("ME").mean().to_period("M")
```

Comment:
Does it matter at all that the order of operations here change? Before the conversion to period happened before the `mean()` call - wonder if there are differences in how those objects implement mean that could cause rounding issues or behavior changes

Reply:
i think the only thing we care about is the `xp` we get here, since we are checking that it round-trips.  i checked manually that this is equivalent to what we have in main